+neurostim/+stimuli/gabor.m

```diff
@@ -164,7 +170,8 @@ function afterFrame(o)
             % Change any or all of the parameters.
             oPhaseSpeed  = o.phaseSpeed;
             if oPhaseSpeed ~=0
-                o.spatialPhase = o.spatialPhase + o.phase + oPhaseSpeed;
+%                 o.spatialPhase = o.spatialPhase + o.phase + oPhaseSpeed;
+                o.spatialPhase = o.spatialPhase + oPhaseSpeed; % increment phase

```

Comment:
Yes, that is a bug fix. 

Please remove the commented out line to avoid confusion down the line.
And you can also remove the unnecessary copies from o.phaseSpeed to oPhaseSpeed and just use o.phaseSpeed

Reply:
Done.

Reply:
Don't merge yet. there's a reference to `o.PhaseSpeed`, but should be `o.phaseSpeed`

Reply:
@bartkrekelberg looks like I was too slow! :)

Reply:
The reference is to oPhaseSpeed, not o.PhaseSpeed?  So it works (but it is confusing to have the local copy and unnecessary in this case)

Reply:
Sorry, I put a spelling mistake in. 
Phasespeed vs phaseSpeed